modules/openapi-generator/src/main/resources/erlang-server/rebar.config.mustache

```diff
@@ -1,6 +1,6 @@
 {deps, [
-    {cowboy, {git, "https://github.com/ninenines/cowboy.git", {tag, "2.5.0"}}},
+    {cowboy, {git, "https://github.com/ninenines/cowboy.git", {tag, "2.8.0"}}},
     {rfc3339, {git, "https://github.com/talentdeficit/rfc3339.git", {tag, "master"}}},
-    {jsx, {git, "https://github.com/talentdeficit/jsx.git", {tag, "2.9.0"}}},
-    {jesse, {git, "https://github.com/for-GET/jesse.git", {tag, "1.5.2"}}}
+    {jsx, {git, "https://github.com/talentdeficit/jsx.git", {tag, "v3.1.0"}}},
```

Comment:
This is a bump in the MAJOR version. According the [CHANGE.md](https://github.com/talentdeficit/jsx/blob/main/CHANGES.md) file from the jsx repository this includes:

> v3.1.0
> - Dialyzer and api spec improvements
> 
> v3.0
> - drop support for OTP versions before 17.0
> - remove definition options for disabling maps globally, {return_maps, false} is still an accepted option to decode/2

AFAIK none of these changes will present an issue. 👍

Reply:
@jfacorro thanks for the review 👍